docs/core-protocol/smart-contracts-api/PublicLock.md

```diff
@@ -822,15 +822,15 @@ Returns the address of the `onKeyPurchaseHook` hook.
 
 | Name | Type | Description |
 |---|---|---|
-| hookAddress | address | address of the hook |
+| hookAddress | address | undefined |
```

Comment:
This change seems backwards, it was clearer before.

Reply:
:+1